encord/dataset.py

```diff
@@ -275,18 +276,70 @@ def add_private_data_to_dataset(
         <tutorials/datasets:Adding data from a private cloud>`
 
         Args:
-            integration_id: str
+            integration_id:
                 EntityId of the cloud integration to be used when accessing those files
             private_files:
                 A str path or Path object to a json file, json str or python dictionary of the files you wish to add
-            ignore_errors: bool, optional
+            ignore_errors:
                 Ignore individual errors when trying to access the specified files
         Returns:
             add_private_data_response List of DatasetDataInfo objects containing data_hash and title
 
         """
         return self._client.add_private_data_to_dataset(integration_id, private_files, ignore_errors)
 
+    def add_private_data_to_dataset_start(
+        self,
+        integration_id: str,
+        private_files: Union[str, Dict, Path, TextIO],
+        ignore_errors: bool = False,
+    ) -> str:
+        """
+        Append data hosted on private clouds to existing dataset.
+
+        This method is only initialising upload job in Encord backend.
+        Once upload job id is returned from this method one can:
+
+         - Forget about next steps, exit terminal,
+           job will continue running in Encord backend without any interruption.
+
+         - Fetch job status with :meth:`add_private_data_to_dataset_get_result`,
+           this could be done in different python session (only upload job id as input matters).
+
+        Args:
+            integration_id:
+                `EntityId` of the cloud integration to be used when accessing those files
+            private_files:
+                A `str` path or `Path` object to a json file, json str or python dictionary of the files you wish to add
+            ignore_errors:
+                Ignore individual errors when trying to access the specified files
```

Comment:
When set to 'true', this will prevent individual errors from stopping the upload process altogether.